index.html

```diff
@@ -22,8 +22,9 @@ <h4 class="time--stat" style="text-align:center">
         time:<span id="timer">0</span>
       </span>
     </h3>
-    <section class="memory-game">
-        <div class="memory-card">
+    <section 
+    class="memory-game">
+        <!-- <div class="memory-card">
           <img class="front-face" src="#" alt="card front face" />
```

Comment:
dont leave the code commented, just leave one of the them , so anyone else can understand the structure